pkgs/development/python-modules/aiogithubapi/default.nix

```diff
@@ -0,0 +1,50 @@
+{ lib
+, aiohttp
+, aresponses
+, async-timeout
+, backoff
+, buildPythonPackage
+, fetchFromGitHub
+, pytest-asyncio
+, pytestCheckHook
+, pythonOlder
+}:
+
+buildPythonPackage rec {
+  pname = "aiogithubapi";
+  version = "21.8.1";
+
+  disabled = pythonOlder "3.8";
+
+  src = fetchFromGitHub {
+    owner = "ludeeus";
+    repo = pname;
+    rev = version;
+    sha256 = "1l59bb956gymi2dkgg88mr0nc80wqvw2dv69qf4f1xr3jxy03qa2";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.py --replace "main" ${version}
```

Comment:
Why is this necessary? I think this deserves an explanation in a comment.

Reply:
We need the the current version in `setup.py`. I updated the `substituteInPlace` to make it clear that's about the version.